src/expressApp.js

```diff
@@ -0,0 +1,7 @@
+const express = require('express')
+const app = express()
+const port = 3000
```

Comment:
Can we have the server for express run on another port than `3000`. That is where the local server for React runs and might cause some issues with the React Application. It would also be nice to add it as a proxy in `package.json`, like in the following tutorial: https://dev.to/loujaybee/using-create-react-app-with-express
```suggestion
const port = 8080
```